### PR TITLE
dev-python/mamba: Add version 0.11.0 for python3_8

### PR DESCRIPTION
**Pull request by ajakk**
All touched packages are maintainer-needed. All tests pass for me except for `dev-python/args`, which doesn't have any tests. Its sole revdep is `dev-python/clint` and those tests pass.

### CODE REVIEW

File: dev-python/mamba/mamba-0.11.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+	>=dev-python/watchdog-0.8.1[${PYTHON_USEDEP}]
+"
+
+python_install() {
+	distutils-r1_python_install
+	python_optimize
+}
+
+python_test() {
+	"${PYTHON}" -m mamba.cli || die "Tests failed under ${EPYTHON}"
```
Comment (juippis) [inline dev-python/mamba/mamba-0.11.0.ebuild:39]: `EPYTHON`.

File: dev-python/mamba/mamba-0.11.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{6,7,8} )
+DISTUTILS_USE_SETUPTOOLS=rdepend
+
+inherit distutils-r1
+
+DESCRIPTION="... testing tool ... Born under the banner of Behavior Driven Development"
```
Comment (juippis) [inline dev-python/mamba/mamba-0.11.0.ebuild:10]: This description could use some cleaning.

File: dev-python/mamba/mamba-0.11.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+HOMEPAGE="http://nestorsalceda.github.io/mamba"
```
Comment (juippis) [inline dev-python/mamba/mamba-0.11.0.ebuild:11]: `https://`

File: dev-python/mamba/mamba-0.11.0.ebuild [hunk 4]
```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{6,7,8} )
```
Comment (juippis) [inline dev-python/mamba/mamba-0.11.0.ebuild:5]: One empty line below EAPI please (not going to ask you to fix other ebuilds for this).

Comment (ajakk) [reply dev-python/mamba/mamba-0.11.0.ebuild]: Might as well do this for all while I'm at it.

**Pull request closed without merging**